README.adoc

```diff
@@ -11,13 +11,22 @@ This project contains the core data models (UUri, UAttributes, etc..) and core s
 
 === How To Build?
 
-The Project uses maven to build the protobuf message. The target to build the code is:
+The Project uses maven/cmake to build the protobuf message. The target to build the code is:
 
+==== How To Build using maven?
 [source,bash]
 ----
 mvn verify -f "pom.xml"
 ----
 
+==== How To Build using cmake?
```

Comment:
@MelamudMichael I tried to pull the project and build it but got the following error below. Obviously I need to pull the protobuf dependencies but instead of installing all the libdev libraries, it would be good if  #72 is merged first:

```
[proc] Executing command: /usr/bin/cmake --no-warn-unused-cli -DCMAKE_BUILD_TYPE:STRING=Debug -DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=TRUE -DCMAKE_C_COMPILER:FILEPATH=/usr/bin/c89-gcc -S/home/czfdcn/MelamudMichael/uprotocol-core-api -B/home/czfdcn/MelamudMichael/uprotocol-core-api/build -G Ninja
[cmake] Not searching for unused variables given on the command line.
[cmake] -- The CXX compiler identification is GNU 11.4.0
[cmake] -- Detecting CXX compiler ABI info
[cmake] -- Detecting CXX compiler ABI info - done
[cmake] -- Check for working CXX compiler: /usr/bin/c++ - skipped
[cmake] -- Detecting CXX compile features
[cmake] -- Detecting CXX compile features - done
[cmake] CMake Error at /usr/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
[cmake]   Could NOT find Protobuf (missing: Protobuf_LIBRARIES Protobuf_INCLUDE_DIR)
[cmake] Call Stack (most recent call first):
[cmake]   /usr/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
[cmake]   /usr/share/cmake-3.22/Modules/FindProtobuf.cmake:650 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
[cmake]   CMakeLists.txt:4 (find_package)
[cmake] 
[cmake] 
[cmake] -- Configuring incomplete, errors occurred!
[cmake] See also "/home/czfdcn/MelamudMichael/uprotocol-core-api/build/CMakeFiles/CMakeOutput.log".
[proc] The command: /usr/bin/cmake --no-warn-unused-cli -DCMAKE_BUILD_TYPE:STRING=Debug -DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=TRUE -DCMAKE_C_COMPILER:FILEPATH=/usr/bin/c89-gcc -S/home/czfdcn/MelamudMichael/uprotocol-core-api -B/home/czfdcn/MelamudMichael/uprotocol-core-api/build -G Ninja exited with code: 1
[main] Building folder: uprotocol-core-api 
[main] Configuring project: uprotocol-core-api 
[proc] Executing command: /usr/bin/cmake --no-warn-unused-cli -DCMAKE_BUILD_TYPE:STRING=Debug -DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=TRUE -DCMAKE_C_COMPILER:FILEPATH=/usr/bin/c89-gcc -S/home/czfdcn/MelamudMichael/uprotocol-core-api -B/home/czfdcn/MelamudMichael/uprotocol-core-api/build -G Ninja
[cmake] CMake Error at /usr/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
[cmake]   Could NOT find Protobuf (missing: Protobuf_LIBRARIES Protobuf_INCLUDE_DIR)
[cmake] Call Stack (most recent call first):
[cmake]   /usr/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
[cmake]   /usr/share/cmake-3.22/Modules/FindProtobuf.cmake:650 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
[cmake]   CMakeLists.txt:4 (find_package)
[cmake] 
[cmake] 
[cmake] Not searching for unused variables given on the command line.
[cmake] -- Configuring incomplete, errors occurred!
[cmake] See also "/home/czfdcn/MelamudMichael/uprotocol-core-api/build/CMakeFiles/CMakeOutput.log".
[proc] The command: /usr/bin/cmake --no-warn-unused-cli -DCMAKE_BUILD_TYPE:STRING=Debug -DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=TRUE -DCMAKE_C_COMPILER:FILEPATH=/usr/bin/c89-gcc -S/home/czfdcn/MelamudMichael/uprotocol-core-api -B/home/czfdcn/MelamudMichael/uprotocol-core-api/build -G Ninja exited with code: 1

```